provider/datagen/src/transform/segmenter/mod.rs

```diff
@@ -821,12 +826,14 @@ impl DataProvider<DictionaryForWordLineExtendedV1Marker> for crate::DatagenProvi
 
 impl IterableDataProvider<DictionaryForWordLineExtendedV1Marker> for crate::DatagenProvider {
     fn supported_locales(&self) -> Result<Vec<DataLocale>, DataError> {
-        Ok(vec![
+        // TODO: Do we actually want to filter these by the user-selected locales? The keys
+        // are more like script selectors...
```

Comment:
Yes, this is the big case where we likely want to return data for locales that aren't necesarily in the user specified set.

We probably should have an additional LSTM-specific flag to allow you to choose which models to build (for which languages, grapheme/codepoint, training fidelity, etc).

Reply:
I guess I'll keep this in for now to preserve behaviour? 

Do we an issue for segmenter datagen locale selection? I couldn't find an open one.

Reply:
Created https://github.com/unicode-org/icu4x/issues/3408